samples/package.json

```diff
@@ -20,7 +20,8 @@
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^3.0.0",
     "chai": "^4.2.0",
-    "mocha": "^6.0.0"
+    "mocha": "^6.0.0",
+    "snap-shot-it": "^7.9.1"
   },
   "scripts": {
     "test": "mocha --timeout=60000"
```

Comment:
we should add the command here:

```
"test:snap": "SNAPSHOT_UPDATE=1 npm test",
```

So that when someone inevitably breaks the output, they can easily update it.